BootloaderCorePkg/BootloaderCorePkg.dsc

```diff
@@ -247,6 +247,24 @@
   gPlatformCommonLibTokenSpaceGuid.PcdUsbKeyboardPollingTimeout | $(USB_KB_POLLING_TIMEOUT)
   gPlatformCommonLibTokenSpaceGuid.PcdLowestSupportedFwVer		| $(LOWEST_SUPPORTED_FW_VER)
 
+  !if $(RSA_SIGN_TYPE) == RSA2048
+    gPlatformCommonLibTokenSpaceGuid.PcdRsaSignType | 0x1
+  !elseif $(RSA_SIGN_TYPE) == RSA3072
+    gPlatformCommonLibTokenSpaceGuid.PcdRsaSignType | 0x2
+  !endif
+
+  !if $(SIGN_HASH_TYPE) == SHA2_256
+    gPlatformCommonLibTokenSpaceGuid.PcdSignHashType | 0x0
+  !elseif $(SIGN_HASH_TYPE) == SHA2_384
+    gPlatformCommonLibTokenSpaceGuid.PcdSignHashType | 0x1
+  !elseif $(SIGN_HASH_TYPE) == SHA2_512
+    gPlatformCommonLibTokenSpaceGuid.PcdSignHashType | 0x2
+  !elseif $(SIGN_HASH_TYPE) == SM3_256
+    gPlatformCommonLibTokenSpaceGuid.PcdSignHashType | 0x3
+  !endif
```

Comment:
Same as RSA_SIGN_TYPE.

Reply:
RSA_SIGN_TYPE and SIGN_HASH_TYPE are different. RSA_SIGN_TYPE has RSA2048/3072. 
SIGN_HASH_TYPE is for hash alg SHA256/384/SM3 alg.